docs/shell-tips/index.md

```diff
@@ -33,3 +33,29 @@ sudo -u $(stat . -c %U) -- wp --path="$PWD" "$@"
 Explanation
 
 The `stat` command returns the owner of the current directory, WordPress root.
+
+## Install and Configure WordPress with WP-CLI
+
+```bash
+wp_install () 
+{ 
+    wp core download --path=$1;
+    cd $1;
+    read -p 'name the database:' dbname;
+    wp core config --dbname=$dbname --dbuser=root --dbpass=awoods --dbhost=localhost;
```

Comment:
You could use `wp core config --prompt` to make it more universal. In a local dev enviroment though, it'll get tedious having to retype the same credentials over and over again. It also speeds up the process a little. Once people copy this to their .bashrc and substitute their credentials, it'll rarely change.